openedx/features/student_certificates/helpers.py

```diff
@@ -192,14 +193,11 @@ def get_verification_url(user_certificate):
     return verification_url
 
 
-def fire_send_email_signal(modulestore, course_key, request, cert):
-    xqueue_send_email = json.loads(request.GET.get('send_email', 'false').lower())
-    if not xqueue_send_email:
-        return
-    course = modulestore().get_course(course_key)
+def fire_send_email_signal(course, cert):
     certificate_reverse_url = certs_api.get_certificate_url(user_id=cert.user.id, course_id=course.id,
                                                             uuid=cert.verify_uuid)
-    certificate_url = request.build_absolute_uri(certificate_reverse_url)
+    domain = Site.objects.get_current().domain
+    certificate_url = ''.join([domain, certificate_reverse_url])
```

Comment:
lets use the constant LMS_ROOT_URL 
and lets just use a + rather than converting into a list and then joining

Reply:
when this is done consider this an approval from my side